vulcanai2/models/basenetwork.py

```diff
@@ -563,6 +566,11 @@ def fit(self, train_loader, val_loader, epochs,
                 fig_number = plt.gcf().number + 1 if plt.fignum_exists(1) else 1
                 plt.show()
 
+            if save_path:
+                time = str(datetime.now())
```

Comment:
you could use your new `get_time()` function here. it may introduce circular dependencies so I would just move that function to model utils

Reply:
oh sorry there should only be one function declaration. if you wanted it to get used across visualization and models, I would recommend it remain in visualization and models can import from the visualization files

Reply:
@rfratila okay, completed